src/pages/project/ProjectEdit.js

```diff
@@ -0,0 +1,75 @@
+import { useCallback } from "react"
+import { useDispatch, useSelector } from "react-redux"
+import { useNavigate, useParams } from "react-router-dom"
+import { REACT_APP_BE_HOST } from "../../config/index"
+import DefaultContainer from "../../layout/DefaultContainer"
+import ContentContainer from "../../layout/ContentContainer"
+import { putProject } from "../../features/project/projectSlice"
+import { Title } from "../../elements/Title"
+import { Button } from "../../elements/Button"
+import ProjectForm from "../../layout/ProjectForm"
+import { validate } from "../../utils/validateFormInput"
+
+function ProjectEdit() {
+  const dispatch = useDispatch()
+  const navigate = useNavigate()
+  const { project } = useSelector((state) => state.project)
+  const defaultValues = {
```

Comment:
Let's memoize `defaultValues`, so we won't recreate object on every component render.